routing/pathfind.go

```diff
@@ -46,6 +46,8 @@ type pathFinder = func(g *graphParams, r *RestrictParams,
 	[]*channeldb.CachedEdgePolicy, float64, error)
 
 var (
+	DefaultEstimator = AprioriEstimatorName
```

Comment:
nit: `DefaultEstimator is the default estimator used when xyz ` (this ones are always hard for me, copilot is your friend here)